packages/docs/src/core/src/spec/transpilation-details.rst

```diff
@@ -12,14 +12,21 @@ Pure Opcodes
 ============
 
 The following opcodes perform stack operations which are constant in terms of L1/L2 state, and do not require modification:
+
 - Arithmetic/pure-math opcodes: 
+<<<<<<< HEAD
+    - ``ADD, MUL, SUB, DIV, SDIV, MOD, SMOD, ADDMOD, MULMOD, EXP, SIGNEXTEND, LT, GT, SLT, SGT, EQ, ISZERO, AND, OR, XOR, NOT, BYT, SHL, SHR, SAAR, SHA3``.
+- \"Pure\" code execution operations: 
+   - ``PUSH1....PUSH32, DUP1...DUP16, SWAP1...SWAP16, POP, LOG0...LOG4, STOP, REVERT, RETURN, PC, GAS, JUMPDEST*``.  \* NOTE: `See section <https://github.com/op-optimism/optimistic-rollup/wiki/JUMP-Transpilation>`_ which involves ``JUMPDEST`` s.
+=======
    - ``ADD, MUL, SUB, DIV, SDIV, MOD, SMOD, ADDMOD, MULMOD, EXP, SIGNEXTEND, LT, GT, SLT, SGT, EQ, ISZERO, AND, OR, XOR, NOT, BYT, SHL, SHR, SAAR, SHA3``.
 - "Pure" code execution operations: 
    - ``PUSH1....PUSH32, DUP1...DUP16, SWAP1...SWAP16, POP, LOG0...LOG4, STOP, REVERT, RETURN, PC, GAS, JUMPDEST*``.  \* NOTE: `See section <https://github.com/op-optimism/optimistic-rollup/wiki/JUMP-Transpilation>`_ which involves ``JUMPDEST``s.
+>>>>>>> newrepo/master
```

Comment:
This looks like an unresolved conflict. Choose one or the other and get rid of the metadata (<<<<<<< HEAD, ========, >>>>>>>).